ConnectedCanteens/ngsild-payloads/Restaurants/restaurantCollery.json

```diff
@@ -77,7 +77,54 @@
             "datasetId": "urn:ngsi-ld:Dataset:Actual:Primaires"
         }
     ],
+    "wasteMeasurement": [
+        {
+            "type": "Property",
+            "value": 25000,
+            "unitCode": "GRM",
+            "observedAt": "2020-06-30T14:00:00Z",
+            "datasetId": "urn:ngsi-ld:Dataset:WasteMeasurement:Vegetables",
+            "wasteType": {
```

Comment:
Maybe we can remove this property since the information is already present in the datasetId.

Reply:
`datasetId` has some information for technical convenience and ease of referencing. but better to keep the metadata for humans :)